source/mapping-rules/mapping-rule-options/time-series.txt

```diff
@@ -0,0 +1,181 @@
+.. _rm-timeseries:
+
+==================================
+Configure a Time Series Collection
+==================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can configure mapping rules to migrate a table that contains a 
+datetime column as a :ref:`time series collection <manual-timeseries-collection>`. 
+Time series collections efficiently store time series data. In time 
+series collections, writes are organized so that data from the same 
+source is stored alongside other data points from a similar point in time.
+
+About this Task
+---------------
+
+- You can use :ref:`field customizations <rm-field-customizations>` to 
+  convert string columns to datetime in Relational Migrator.
+
+- CDC jobs that have time series configurations do not create the 
+  time series collections until the CDC job completes. 
+
+- Compared to normal collections, storing time series data in time series
+  collections improves query efficiency and reduces the disk usage for
+  time series data and :term:`secondary indexes <secondary index>`.
+
+- Use cases for time series collections include 
+  :abbr:`IoT (Internet of Things)`, forecasting, and time based analysis.
+
+Before you Begin
+----------------
+
+- For full details on each time series field component, see 
+  :ref:`time-series-fields`.
+
+- If the time series data in your sync job does not require any 
+  complex transformations you may benefit from migrating your time 
+  series data as a separate sync job. This can make it easier to 
+  preload a large time series collection and once completed migrate 
+  the rest of your data.
+
+- If your migrating larges amounts of data, consider using 
+  :ref:`table filters <rm-table-filters>` to batch your sync jobs 
+  based on logical conditions. For example, you might import one year 
+  of historical data at a time using table filters. This allows a large 
+  job to be broken into more manageable segments.
+
+- If a long-running sync job fails unexpectedly (for example, due to a 
+  database outage) and is not able to automatically recover, you may 
+  be able to use :ref:`table filters <rm-table-filters>` to create a 
+  new sync job that picks up from where the previous one left off. This 
+  approach saves you from re-importing your previous data from scratch.
```

Comment:
Consider rewording "from scratch" to avoid idiomatic language.

```suggestion
  approach lets you avoid having to repeat the entire sync job.
```